src/sync.py

```diff
@@ -62,12 +62,16 @@ def sync():
                 )
                 if not api.requires_2sa:
                     if "drive" in config and enable_sync_drive:
+                        LOGGER.info("Syncing drive...")
```

Comment:
I've decided to add logs at `info` level here - without them it's hard to see if anything happens.
They aren't as "spammy" as debug level.